react-frontend/src/components/Value/Value.jsx

```diff
@@ -1,32 +1,31 @@
-import React, { useMemo } from 'react'
-import { format } from "date-fns";
-import './value.css';
-
-const Value = ({title, value, unit, timestamp, state, level}) => {
+import React, { useMemo } from "react";
+import "./value.css";
 
+const Value = ({ title, value, unit, state, level }) => {
   const renderState = useMemo(() => {
-    if(state === 'error') {
-    return "error"
-  } else if (state === 'loading' || !value || !timestamp) {
-    return "loading"
-  } else {
-    return "success"
-  }
-  }, [state, value, timestamp])
+    if (state === "error") {
+      return "error";
+    } else if (state === "loading" || !value) {
+      return "loading";
+    } else {
+      return "success";
+    }
+  }, [state, value]);
 
   return (
     <div className="metric">
       <h1 className="title">{title}</h1>
       {renderState === "error" && <p className="error">An error occured.</p>}
-      {renderState === "loading" && <p>Loading...</p>}
+      {renderState === "loading" && <p className="center-text">Loading...</p>}
       {renderState === "success" && (
```

Comment:
These states are also pretty good to write tests for. I.e. making sure that the correct UI elements render based on `renderState`. However, we can take a look at that in a new PR.